src/Sql/dbo/Functions/UserCipherDetails.sql

```diff
@@ -23,6 +23,13 @@ SELECT
         THEN 1
         ELSE 0
     END [Edit],
+    CASE
```

Comment:
Need to use the rule of specificity that we came up with earlier in Slack.
```sql
COALESCE(CU.[ReadOnly], CG.[ReadOnly], 0) AS [ReadOnly],
COALESCE(CU.[HidePasswords], CG.[HidePasswords], 0) AS [HidePasswords]
```

Reply:
This one is the inverse.

Reply:
Had to use a Case here to invert it, not sure if there's a better way but `~COALESCE(CU.[HidePasswords], CG.[HidePasswords], 0) AS [ViewPassword]` did not work.